### PR TITLE
Fix issue 2204 in tour - blocking file and edit menu

### LINKED ISSUE #2204: Windows: something is preventing "File" and "Edit" menus being clicked
See this screencast:

spyder-menu-bug

When hovering over the File and Edit menu items (and the left half of the Search menu item), the item does not raise, and clicking on it doesn't open the menu. However, once opening the menu items further to the right, by hovering over left the File and Edit menus also open.

This kind of problem is described [here](http://qt-project.org/forums/viewthread/9043/) and is apparently due to some widget being parented by MainWindow incorrectly.  I have not yet found which widget is causing this.

### PR DESCRIPTION
**Pull request by goanpeca**
Fixes #2204

### CODE REVIEW

**Comment by goanpeca** [comment]
github opens an issue for every pull request... so can I associate a pull request for an already existing issue?.. cause now Issue 2208 solves issue 2204... it is confusing

File: spyderlib/tour.py [hunk 1]
```diff
@@ -810,13 +810,17 @@ def __init__(self, parent):
...
 
+        # Widgets
         self.canvas = FadingCanvas(self.parent, self.opacity_canvas,
                                    self.duration_canvas, self.easing_curve,
                                    self.color)
         self.tips = FadingTipBox(self.parent, self.opacity_tips,
                                  self.duration_tips, self.easing_curve)
 
-        # signals and slots
+        # Widgets setup
```
Comment (ccordoba12) [inline spyderlib/tour.py]: Please add a note here that the next line is needed to fix issue #2204

**Comment by goanpeca** [comment]
Ok will do, question, do you think the tours will need any config? should it become a plugin/dockwidget?

I do not think so, but just wondering

**Comment by goanpeca** [comment]
@ccordoba12 done

**Comment by ccordoba12** [comment]
Ok, thanks!

**Comment by Nodd** [comment]
This is the same problem I had in https://github.com/Nodd/spyder_autopep8/pull/13, and I think the fix over there is better, instead of rendering it transparent, it is not there at all.
BTW there should be either an example or even better a base class for plugins without a dockwidget.

**Comment by goanpeca** [comment]
Yes I agree, but making it transparent was simpler to do :p

**Pull request merged**